src/tool_fsharp/tool_fsc.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.tasks" />
+
+  <PropertyGroup>
+    <VersionPrefix>$(CliVersionPrefix)</VersionPrefix>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <PublishDir>$(FSharpDirectory)</PublishDir>
+    <VersionSuffix>$(CommitCount)</VersionSuffix>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NetCore.App" Version="$(CLI_SharedFrameworkVersion)" />
+    <PackageReference Include="Microsoft.FSharp.Core.netcore" Version="$(CLI_FSharp_Version)" />
+    <PackageReference Include="Microsoft.FSharp.Compiler.netcore" Version="$(CLI_FSharp_Version)" />
```

Comment:
Just going to repeat @ellismg 's comment that github is now hiding because it's on the older version of the file: 

> What repository builds these packages and how can I replicate that?